### ISSUE TITLE
 Search engine shows Angular.js template for webmaker zh-tw website

### ISSUE THREAD

**Issue by alicoding** [mozilla/webmaker.org#1157]
Steps to reproduce:

Searching webmaker in Google in Traditional Chinese.

Actual results:

The Angular.js template shows up instead of the translated content, as shown in https://www.google.com.tw/search?q=webmaker

Expected results:

The website content should be generated instead of the template text.

**Comment by irvin** [comment]
![2015-01-22 10 19 49](https://cloud.githubusercontent.com/assets/165936/5849398/3c4f27fe-a220-11e4-8a42-fe2307c07d5c.png)
still a valid bug, I'm not sure if it's happen to another locale

**Comment by alicoding** [comment]
@jdotpz do you know if we still have that prerender service up? I think @gvn might know a bit more about `prerender` but I'm just not sure why it's not working as expected now.

**Comment by jdotpz** [comment]
AFAIK, it's still up.  https://bugzilla.mozilla.org/show_bug.cgi?id=984532 tells me that we didn't add any DNS for it, and hosted it on Heroku via @gvn 
http://wm-prerender.herokuapp.com/ is it's url, and I just bounced that service.

**Comment by alicoding** [comment]
@jdotpz I'm not sure if that's what I'm expecting to see, but it is taking forever and also it's a blank HTML (viewed the source code). I honestly have less knowledge about this.

The real question here would be. Did we even set this up for Webmaker.org? Or just event?

**Comment by gvn** [comment]
I believe we only set this up for events.webmaker.org. @k88hudson may have set it up for wm.org though...

**Comment by alicoding** [comment]
Yeah this is weird. https://github.com/mozilla/webmaker.org/blob/master/app.js#L139 I can see it here, but why this is not rendering properly O_o

**Comment by cadecairos** [comment]
![screen shot 2015-02-18 at 15 27 21](https://cloud.githubusercontent.com/assets/438003/6256049/df71526e-b782-11e4-8c91-8a2267d0d5b8.png)

**Comment by thisandagain** [comment]
Current search results for "Webmaker" in `en-US`:
![screen shot 2015-03-16 at 4 29 49 pm](https://cloud.githubusercontent.com/assets/747641/6678600/c873942e-cbf9-11e4-99ba-ef6ba07dcea7.png)

**Comment by edrushka** [comment]
@thisandagain I'm closing the Misc dev ticket. Just reminding you to move or label this as you see fit so it doesn't get lost.

**Comment by alicoding** [comment]
This is pretty bad even for other social network site display this thing as well... I'm not sure what happen to prerender we are using here?

![vyu5xb9-](https://cloud.githubusercontent.com/assets/4368149/7004677/3a9200d8-dc3d-11e4-8400-3955567547ec.png)
